### ISSUE TITLE
How to make reset button to exit any game without hotkey?

### ISSUE THREAD

**Issue by rejuas** [RetroFlag/retroflag-picase#28]
Im using recalbox and my quiestion is : there is any way to modify this script for quit or exit any game emulation pressing only the reset button?
anyone can make it for me?

**Comment by codecomp** [comment]
If i think you mena what I think you mean this rewrite will do what you want

https://github.com/crcerror/retroflag-picase/

**Comment by slappymcphee** [comment]
> If i think you mena what I think you mean this rewrite will do what you want
> 
> https://github.com/crcerror/retroflag-picase/

can someone install this right over the original install from retroflag?

**Comment by crcerror** [comment]
yes just write install with `wget -O - "https://raw.githubusercontent.com/crcerror/retroflag-picase/master/install.sh" | sudo bash` and you will be fine.
